src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_rsa.c

```diff
@@ -16,15 +18,15 @@ PALEXPORT RSA* CryptoNative_RsaCreate()
     return rsa;
 }
 
-PALEXPORT int32_t CryptoNative_RsaUpRef(RSA* rsa)
+PALEXPORT int32_t AndroidCryptoNative_RsaUpRef(RSA* rsa)
 {
     if (!rsa)
         return FAIL;
     rsa->refCount++;
```

Comment:
I can't leave a comment in the right place, but: this doesn't look like an atomic int to me.